src/schema.ts

```diff
@@ -234,45 +234,34 @@ function timeUnitSummary(unit: TimeUnit, summary: Summary, type: Type) {
       });
   }
 
-  return binStats(bin, summary, type);
+  return binStats(bin, summary);
 }
 
 /**
  * @return a new summary based on a new binning scheme and old summary statistics
  */
-function binStats(bin, summary: Summary, type: Type): Summary {
+function binStats(bin, summary: Summary): Summary {
   // have the timeUnit bin scheme, need to determine new stats
   // start with summary, pre-binning
   const result = extend({}, summary);
 
-  const newUnique = binUnique(bin, summary.unique);
-  const values = uniqueToValues(newUnique);
-
-  result.unique = newUnique;
-  // 'count', 'valid', and 'missing' don't change
-  result.distinct = (type === Type.ORDINAL) ? keys(result.unique).length : (bin.stop - bin.start) / bin.step;
```

Comment:
Now I recall why you add type.  It's actually not `type`, but rather `scaleType`.  But I guess let's ignore those for now.